src/main/java/javaClasses/homework_4/stepan_alvajyan/Calculations.java

```diff
@@ -0,0 +1,24 @@
+package javaClasses.homework_4.stepan_alvajyan;
+
+public class Calculations {
+
+    double getMean(int a, int b) {
+        return (a + b) / 2;
+    }
+
+    double getMean(int a, int b, int c) {
```

Comment:
You are providing int arguments to a method which returns double. It's fine to have implementation with double precision, it is not always that mean of integer numbers is mean as well. But current task requires to have arguments and return type to be the same. So please change)